packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -313,6 +313,26 @@ export function findSelectTokenButton(
     .should('have.text', text)
 }
 
+export function checkForCustomDestinationAddressInTransactionDetail(
+  customAddress: string
+): Cypress.Chainable<JQuery<HTMLElement>> {
+  cy.findAllByLabelText('Transaction details button')
+    .first()
+    .click()
+    .then(() => {
```

Comment:
I'm not sure this is necessary (the `.then` after a `.click`)? would be good to try out

Reply:
`then` is not necessary and even cause issues in some case: https://docs.cypress.io/guides/core-concepts/retry-ability#Incorrectly-checking-if-the-stub-was-called